### PR TITLE
FEATURE: SD-103 Documentation for Combo chart

### PR DESCRIPTION
**Pull request by nadtan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/43/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/57/ : SUCCESS in 1m 10s

**Comment by evgeniabogranskaya** [comment]
@nadtan , is is ready for my review, please?

**Comment by nadtan** [comment]
> @nadtan , is is ready for my review, please?

Yes, it is. please help me review it :)

File: docs/combo_chart_component.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+---
+title: Combo Chart
+sidebar_label: Combo Chart
+copyright: (C) 2019 GoodData Corporation
```
Comment (evgeniabogranskaya) [inline docs/combo_chart_component.md]: In other topics, it says:
copyright: (C) 2007-2018 GoodData Corporation

Pls check for consistency, thanks.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/48/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/62/ : SUCCESS in 1m 14s

**Comment by evgeniabogranskaya** [comment]
Proofread: https://github.com/gooddata/gooddata-ui/pull/222/commits/dff02da8ee633bea72a8552ae83b97aec62412d8 + one comment to resolve
Otherwise, LGTM

File: docs/combo_chart_component.md (hunk 1 above)
Comment (macekond) [reply docs/combo_chart_component.md]: I checked https://confluence.intgdc.com/display/pap/GoodData+Copyright+Instructions and it seems it is correct

Comment (evgeniabogranskaya) [reply docs/combo_chart_component.md]: Cool, thanks!

**Comment by nadtan** [comment]
Hi @ivan-nejezchleb , Please help me review on my last commit. I have fixed the documentation base on your comments. 
Thank you

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/50/ : SUCCESS in 48s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/66/ : SUCCESS in 2m 16s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/51/ : SUCCESS in 5m 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/67/ : SUCCESS in 3m 37s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/52/ : SUCCESS in 1m 39s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/68/ : SUCCESS in 2m 15s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/69/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/20/ : SUCCESS in 20s

**Pull request merged**